src/gwt/src/org/rstudio/studio/client/workbench/views/files/model/FilesServerOperations.java

```diff
@@ -74,6 +74,10 @@ void moveFiles(ArrayList<FileSystemItem> files,
    void renameFile(FileSystemItem file,
                    FileSystemItem targetFile,
                    ServerRequestCallback<Void> serverRequestCallback);
+   
+   // touch file
```

Comment:
Nit: this comment doesn't add anything more than the function's name 
```suggestion
```

Reply:
This was done for consistency with the other comments for the other functions.

I will remove this, because I also agree that the function name is self-explanatory. Unless I hear otherwise, I won't remove the other comments -- only this one.